### PR TITLE
Docs: utils.ts of i18n

### PR DESCRIPTION
**Pull request by MaxCar31**
chore documentation of utils.ts

### CODE REVIEW

File: src/i18n/config.ts [hunk 1]
```diff
@@ -1,6 +1,13 @@
-// src/i18n/config.ts
+
+
```
Comment (ferjozsot23) [inline src/i18n/config.ts]: Deñete 1 unuseful space

**Review by ferjozsot23 — CHANGES_REQUESTED** [review]
Good work!

**Review by ferjozsot23 — APPROVED** [review]
Nice!

**Review by juanfcarrillo — APPROVED** [review]
Ola

**Pull request merged**